static/src/stylesheets/layout/nav/_my-account.scss

```diff
@@ -0,0 +1,52 @@
+.my-account {
+    .dropdown-is-open & {
+        @include mq($until: tablet) {
+            z-index: $zindex-main-menu + 1;
+        }
+    }
+}
+
+.my-account--icon {
+    svg {
+        height: 18px;
+        width: 18px;
+
+        @include mq(mobileMedium) {
+            height: 23px;
+            width: 23px;
+        }
+
+        path {
```

Comment:
I think if you remove the colour from the path in the svg, you then don't have to nest the `fill` in path